application/templates/cms/edit_measure_version.html

```diff
@@ -64,9 +64,11 @@
 
 
                 {% if measure_version.status == 'APPROVED' and current_user.can(CREATE_VERSION) %}
```

Comment:
@TheDoubleK maybe we should unnest these `if` conditions, so that:

1. If approved and latest and current user can create version then show update button.
2. If approved and current user can publish then show unpublish button

Reply:
I've un-nested some ifs in a new commit.

Reply:
👍